### DISCUSSION TITLE
Pair Programming Opportunity!

### DISCUSSION THREAD

**Discussion by seandenigris** [PortingPDFtalk/PharoPDFtalk#2] [General]
Per @ChristianHaider [on the Pharo Users ML, 2022-05-23](https://lists.pharo.org/empathy/thread/FIV6XT3XYL42VX44UYHCIP5WHDPDV42A?hash=FIV6XT3XYL42VX44UYHCIP5WHDPDV42A#FIV6XT3XYL42VX44UYHCIP5WHDPDV42A):
> Now I am looking for a Pharoer with an interest in PDF and/or the 
transformation tool and some time at hand to pair up for some coding. This 
is an exciting project. Heads up - who wants to be part of this?

**Comment by seandenigris** [comment]
I am interested. 

Cross-platform support is something important to me. As the de facto maintainer of Magritte, I'd love for its latest version to be continually available on non-Squeak variants. I also remember being frustrated that the version of Xtreams on Pharo was lagging behind and having no idea how to update the port from VW. @dalehenrich and I worked on something in this area at ESUG one year. I remember we even had a basic POC doing something useful.

**Reply by ChristianHaider** [reply]
Thank you for the nice session the other day. All issues fixed and published. 
Any time again...

**Comment by seandenigris** [comment]
Yes, it was fun and we should do it again for sure
